packages/common/src/hardhat/tasks/xchainSetup.ts

```diff
@@ -1,11 +1,33 @@
 import { Deployment } from "hardhat-deploy/types";
+import { interfaceName } from "../../Constants";
 import { task } from "hardhat/config";
 import { Contract } from "web3-eth-contract";
 import { CombinedHRE } from "./types";
 import Web3 from "web3";
 const { utf8ToHex, toBN } = Web3.utils;
 const assert = require("assert");
 
+// Note: Make sure that the following chain names are all lower case.
+const L2_CHAIN_NAMES: { [key: string]: number } = {
+  arbitrum: 42161,
+  optimism: 10,
+  boba: 288,
+  polygon: 137,
+};
+const getChainNameForId = (chainId: number): string => {
```

Comment:
you need to merge master. this script changed somewhat with the OP deployment. this function will be useful in re-factoring the `setupOvmBasedL1Chain` function that is currently in master. it does the exact same thing with `capitalizeFirstLetter`

Reply:
Nice done